### ISSUE TITLE
Outdated, then not showing up

### ISSUE THREAD

**Issue by Xyrophobia** [Haplo064/SlideCast#11]
the plugin was shown as Outdated, now after i deleted it for trying a reinstall its not showing up in the Plugin Installer at all anymore? same with the jobicons and the chat translator plugin for me.
While i do see the Chat Bubbles one in the Plugin Installer

**Comment by JaimieKnox** [comment]
Yeah xlplugins got an overhaul recently and it broke some of the plugins. I'm hoping the creator updates this to make it compatible.

**Comment by HitagiSenjougaharaFFXIV** [comment]
You have slidecast now in "Simple Tweaks" - UI Tweaks - Cast Bar Adjustments then click Show Slidecaste Marker.
You Welcome

**Comment by Xyrophobia** [comment]
Yes i just found that through a streamer an hour ago aswell. thank you tho Hitagi :)

**Comment by JaimieKnox** [comment]
Yeah I just preferred having a simple vertical line instead of the rounded box. It'll probably grow on me though. Thanks :)
